### PR TITLE
Calculate session durations in seconds

### PR DESCRIPTION
**Pull request by Amokrane**

### CODE REVIEW

File: publisher-sdk/src/main/java/com/criteo/publisher/Session.kt [hunk 1]
```diff
@@ -21,9 +21,9 @@ internal class Session(private val clock: Clock) {
...
 
   /**
-   * Returns the current time in milliseconds since the initialization of the SDK.
+   * Returns the current time in seconds since the initialization of the SDK.
    *
-   * @return the difference, measured in milliseconds, between the current time and the SDK init.
+   * @return the difference, measured in seconds, between the current time and the SDK init.
    */
-  fun getDuration(): Long = clock.currentTimeInMillis - startingTime
+  fun getDurationInSeconds(): Long = (clock.currentTimeInMillis - startingTime) / 1000
```
Comment (TurpIF) [inline publisher-sdk/src/main/java/com/criteo/publisher/Session.kt:28]: Nit: I guess we can use a simple `Int` then

**Review by TurpIF — COMMENTED** [review]
Didn't you forget to change the `ContextProvider` class ?

File: publisher-sdk/src/main/java/com/criteo/publisher/Session.kt (hunk 1 above)
Comment (Amokrane) [reply publisher-sdk/src/main/java/com/criteo/publisher/Session.kt]: Done

File: publisher-sdk/src/androidTest/java/com/criteo/publisher/SessionTest.kt [hunk 2]
```diff
@@ -35,11 +37,11 @@ class SessionTest {
...
 
-    whenever(clock.currentTimeInMillis).thenReturn(1379L)
+    whenever(clock.currentTimeInMillis).thenReturn(2337L)
 
     val session = DependencyProvider.getInstance().provideSession()
-    val duration = session.getDuration()
+    val duration = session.getDurationInSeconds()
 
-    assertThat(duration).isEqualTo(42L)
+    assertThat(duration).isEqualTo(1L)
```
Comment (TurpIF) [inline publisher-sdk/src/androidTest/java/com/criteo/publisher/SessionTest.kt:45]: As it is an integer, `1` is sufficient

File: publisher-sdk/src/androidTest/java/com/criteo/publisher/SessionTest.kt [hunk 3]
```diff
@@ -21,9 +21,11 @@ import com.nhaarman.mockitokotlin2.spy
 import com.nhaarman.mockitokotlin2.whenever
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Test
+import kotlin.time.ExperimentalTime
 
 class SessionTest {
 
+  @ExperimentalTime
```
Comment (TurpIF) [inline publisher-sdk/src/androidTest/java/com/criteo/publisher/SessionTest.kt:28]: What is this used for ?

Comment (Amokrane) [reply publisher-sdk/src/androidTest/java/com/criteo/publisher/SessionTest.kt:28]: oops left over from when I was playing with `Duration` experimental APIs. Removed.

**Review by TurpIF — APPROVED** [review]

**Review by TurpIF — APPROVED** [review]

**Pull request merged**